report_qr/models/ir_actions_report.py

```diff
@@ -0,0 +1,35 @@
+import io
+
+import qrcode
```

Comment:
i think qrcode should be added as a external library dependency , and perhaps in the repo requirements. https://github.com/OCA/reporting-engine/blob/14.0/requirements.txt
unless there isn't some indirect dependency that guarantees qrcode to be installed?